### DISCUSSION TITLE
FYI - slack (globally) is down.

### DISCUSSION THREAD

**Discussion by iamtrask** [OpenMined/courses#128] [Announcements]
For anyone trying to use OpenMined's slack today - it is down as are most other slacks. Please just drop your questions and queries here.

**Comment by MantisClone** [comment]
Hi, 10 days later, I'm still unable to join the OpenMined Slack.  Is it still down?

**Reply by yemikifouly** [reply]
Hello David! 👋  I am so sorry to hear that. How are you trying to join? By using https://slack.openmined.org? Or some other way?

**Reply by MantisClone** [reply]
Yes, when I click that link i get "This link is no longer active. To join this workspace, you’ll need to ask the person who originally invited you for a new link."

**Reply by yemikifouly** [reply]
Oh no! Let me have that fixed and I will get back to you David! 🙂

**Reply by yemikifouly** [reply]
@DMats It is working now. Do you mind trying again? 🙂

**Reply by MantisClone** [reply]
It worked! Thank you.

**Reply by lucidprojects** [reply]
The invite link looks like it is down again. 

I get

```
Invitation expired
It looks like you’re trying to accept an invitation, but the link was deactivated.

Contact the person who originally invited you about getting a new link or manually inviting you.
```

Are we still able to join the slack?
